HACKING.md

```diff
@@ -16,7 +16,31 @@ and then update `deploy/operator.yaml` to point to your images and follow rest o
 ```
 
 3. Tag and push both images to quay.io (or a container registry of your choice). Make sure that images are publicly available.
-4. Since we will be going to test with our version of images, we need to modify CSV file to point to our version of image. This can be done by modifying following file:
+
+## Automatic creation of bundle and index image
+
+Assuming we have `opm` command and images of `local-storage-operator` and `local-diskmaker`, we can use following command to create the bundle and index image:
+
+
+```
+~> ./hack/sync_bundle -o <operator_image> -d <diskmaker_image> -b <bundle_image> -i <index_image> bundle
+
+~> ./hack/sync_bundle -o quay.io/openshift/origin-local-storage-operator:latest  \
+        -d quay.io/openshift/origin-local-storage-diskmaker:latest \
+        -b quay.io/gnufied/local-storage-bundle:v1 \
+        -i quay.io/gnufied/gnufied-index:v1 bundle
+~> docker push quay.io/gnufied/gnufied-index:v1
+```
+
+This should give us index image `quay.io/gnufied/gnufied-index:v1` which we can use in `examples/olm/catalog-create-subscribe.yaml` file:
```

Comment:
Personal nitpick - I'd remind that the `examples/olm/catalog-create-subscribe.yaml` file needs to be updated before we create it to use the index image. So maybe:

> This should give us index image `quay.io/gnufied/gnufied-index:v1`.
> Update the `CatalogSource` entry in `examples/olm/catalog-create-subscribe.yaml` to point to your newly created index image. Once updated, we can create this without further changes:
> ```
> ~> oc create -f examples/olm/catalog-create-subscribe.yaml
> ```

Reply:
done